libraries/anaconda35/Dockerfile

```diff
@@ -0,0 +1,25 @@
+#  $ docker build . -t continuumio/anaconda3:latest -t continuumio/anaconda3:5.3.0
+#  $ docker run --rm -it continuumio/anaconda3:latest /bin/bash
+#  $ docker push continuumio/anaconda3:latest
+#  $ docker push continuumio/anaconda3:5.3.0
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
+
+RUN apt-get update --fix-missing && apt-get install -y wget bzip2 ca-certificates \
```

Comment:
Any reason we can't move the `curl`, `grep`, `sed`, and `dpkg` packages up here and then follow the [`rm pattern`](https://github.com/algorithmiaio/langpacks/blob/master/libraries/java11/Dockerfile#L21).

Reply:
Done, good call